microsoft/iot-cardboard-js
# Visual rules form panel - UI only

### Summary of changes 🔍 
In this PR I created the components that will display the Visual rules left panel. In an effort to keep the pull request short I created only the UI portion of these components. All data and types will be defined more thoroughly in the next pull request. If there are any comments on designs or panel navigation, please let me know. 

There are some temporary things created to allow this to work correctly. In the code, some mock data is present. In the UI, there is a button floating to allow testers access to this functionality. These will be removed in future pull requests.

### Testing 🧪
Flight local storage key, to be able to see the functionality:
- Open developer tools of the browser and select application tab.
- Open localstorage variables
- Add this key: `cardboard.feature.visualRulesFeature` and set its value to `true`
- Refresh the page
- OR -
- Open browser dev tools and run this in console: `localStorage.setItem('cardboard.feature.visualRulesFeature',true);`

To test:
- Open a scene and navigate to behaviors form
- Click on visual rules button
- Test out the UI for visual rules form

### Checklist ✔️
